### PR TITLE
Draggable meeting documents

### PR DESCRIPTION
**Pull request by webelaine**
Inside the Meeting Documents metabox, which appears in the Editor's Inspector Panel for the Council Meeting post type:
- Make individual documents draggable for easy re-sorting (previously were not draggable)
- Enable adding variable numbers of documents (previously capped at 5)

I'm not sure the table format - showing the title next to the URL - is ideal, but it's functional. It might be worth revisiting formatting so the title appears above the URL, as in the previous non-draggable version, so a few more characters of each input are visible.

Addresses https://github.com/happyprime/pcouncil.org/issues/208

### CODE REVIEW

File: js/metabox-draggable.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	 */
+	$("#dragdrop-sortable").sortable({
+		handle: ".dragdrop-sortable-item-header",
+		placeholder: "dragdrop-highlight"
+	});
+
+    /**
+     * Update numeric order
+     */
+    function dragdropChnageItemOrder(){
```
Comment (philcable) [inline js/metabox-draggable.js]: Just a small typo - the `a` and `n` got transposed, I suggest a quick search and replace :)
```suggestion
    function dragdropChangeItemOrder(){
```

File: js/metabox-draggable.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    	var allDragdropItems = $('.dragdrop-sortable-content li.single-sortable-item');
+    	$.each(allDragdropItems, function(index, value){
+            var childItem = allDragdropItems.eq(index);
+            childItem.find('input.dragdrop-set-order').val(index);
+    		childItem.find('span.dragdrop-item-count').html(index+1);
```
Comment (philcable) [inline js/metabox-draggable.js]: Not a big deal at all, just seeing this small spacing inconsistency. I have [this ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) for VSCode that shows issues and errors inline, which is super nice and highly recommended! It also flags things like whether double or just single quotes are needed (which is probably the case in the `sortupdate` function below).

I also know we don't have any ESLint configuration set up on this repo 🙃, but the extension can be configured with a global fallback, which is also very cool.

**Review by philcable — COMMENTED** [review]
Looking great! Just one typo - and some super minor syntax/spacing issues that we don't necessarily need to worry about right now - in the `js/metabox-draggable.js` file.

File: js/metabox-draggable.js (hunk 2 above)
Comment (webelaine) [reply js/metabox-draggable.js]: Ah, the ugly results of copied and pasted code! Thanks for your sharp eyes - I've updated both the typo and the spacing. I have that ESLint extension but I must not have something set quite right as it isn't flagging quotes.

**Pull request merged**